packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -111,7 +111,7 @@ class ProductUIGet(OutputSchema):
 ExtraCreditsUsdRangeInt: TypeAlias = Annotated[int, Field(ge=0, lt=500)]
 
 
-class GenerateInvitation(InputSchema):
+class InvitationGenerate(InputSchema):
```

Comment:
should this not be InvitationCreate ??
and the one after is InvitationCreated or InvitationGet ?

Reply:
The operation is called generate, not create

![image](https://github.com/user-attachments/assets/1f28d065-3327-49b6-9c1c-aa01018443c9)